src/pyodide/BUILD.bazel

```diff
@@ -229,27 +235,41 @@ data = wd_ts_bundle_capnp(
         "pyodide-bucket.json@rule",
     ],
     lint = False,
-    modules = ["python-entrypoint-helper.ts"],
+    modules = MODULES,
     schema_id = "0xbcc8f57c63814005",
     tsconfig_json = "tsconfig.json",
 )
 
-cc_capnp_library(
-    name = "pyodide",
-    srcs = ["pyodide.capnp"],
-    data = data,
-    include_prefix = "pyodide",
-    strip_include_prefix = "",
-    visibility = ["//visibility:public"],
-    deps = ["@workerd//src/workerd/jsg:modules_capnp"],
-)
-
 genrule(
     name = "pyodide.capnp.bin@rule",
     srcs = [
-        "pyodide.capnp",
+        ":pyodide.capnp",
         "//src/workerd/jsg:modules.capnp",
-    ] + data,
+    ] + [
```

Comment:
fyi @hoodmane this could probably be cleaned up, but I opted to keep it very explicit since I'm not entirely sure what's going on. If this is a common pattern, then maybe we need to add helper transformation function to the bundle .bzl, but returning values from macros is not a common pattern.